AWSAppSyncClient/AWSAppSyncClientConfiguration.swift

```diff
@@ -365,6 +384,8 @@ public class AWSAppSyncClientConfiguration {
                                                                urlSessionConfiguration: urlSessionConfiguration,
                                                                authProvider: oidcAuthProvider)
         }
+        
+        networkTransport.retryStrategy = retryStrategy
```

Comment:
pass the retry strategy as part of `makeNetworkTransport...` method family to keep all the assignments together